task_executor/src/task_executor/base_executor.py

```diff
@@ -326,7 +371,7 @@ def prepare_task(self, task):
 
         expected_nav_duration = rospy.Duration(0)
         if self.active_task.start_node_id != '':                    
-            expected_nav_duration = self.expected_navigation_duration(task)
+            expected_nav_duration = self.expected_navigation_duration_now(self.active_task.start_node_id)
             rospy.loginfo('expected_nav_duration:  %s' % expected_nav_duration.to_sec())
```

Comment:
Currently  navigation and timing of navigation are separate smach states that are constructed together, then navigation is actually triggered when the state machine is run. So I need it beforehand for the current system.